File: test/cs/case10_not.cs
using System.Collections.Generic;
using System.IO;
using System;

public static class Program
{
    public static bool invert(bool flag)
    {
        if ((!flag))
        {
            return true;
        }
        else
        {
            return false;
        }
    }

    public static void Main(string[] args)
    {
        Pytra.CsModule.py_runtime.print(invert(false));
    }
}

File: test/rs/case10_not.rs
#[path = "../../src/rs_module/py_runtime.rs"]
mod py_runtime;
use py_runtime::{perf_counter, py_in, py_len, py_print, py_slice};

// このファイルは自動生成です（native Rust mode）。

fn invert(flag: bool) -> bool {
    if (!flag) {
        return true;
    } else {
        return false;
    }
}

fn main() {
    py_print(invert(false));
}
